rs/execution_environment/src/execution_environment.rs

```diff
@@ -2714,12 +2714,18 @@ impl ExecutionEnvironment {
     ) -> Result<(), UserError> {
         let nodes = args.get_set_of_nodes()?;
         let registry_version = args.get_registry_version();
+
+        let key_id = args
+            .key_id
+            .try_into()
+            .map_err(|err| UserError::new(ErrorCode::CanisterRejectedMessage, err))?;
```

Comment:
@dfinity/execution : Which error could would be the most appropriate here?

Reply:
`ErrorCode::CanisterRejectedMessage` is fine.